src/ActionList.jsx

```diff
@@ -19,6 +20,31 @@ export default class ActionList extends Component {
 
   componentDidMount() {
     this.scrollToBottom(true);
+    if (!this.props.draggableActions) return;
+    const container = ReactDOM.findDOMNode(this.refs.rows);
+    this.drake = dragula([container], {
+      copy: false,
+      copySortSource: false,
+      mirrorContainer: container,
+      accepts: (el, target, source, sibling) => (
+        !sibling || parseInt(sibling.getAttribute('data-id'))
+      ),
+      moves: (el, source, handle) => (
+        parseInt(el.getAttribute('data-id')) &&
+        handle.className.indexOf('selectorButton') !== 0
```

Comment:
Did you mean `!== -1`?
Also, using className here looks unreliable. Can we use `data` attribute or something? (which is also not great, but better)